stripped/colabs/Regresja_logistyczna_i_SVM_stripped.py

```diff
@@ -0,0 +1,1180 @@
+
+OLS_report_and_ANOVA(summer_data, 'pustynia', 'SoilM_100_200cm')
+
+perform_Kruskal_Wallis(summer_data, 'pustynia', 'SoilM_100_200cm', 13)
+
+test_for_residuals_normality(summer_data, 'P/PET')
+
+plot_density(summer_data, 'pustynia', 'P/PET', random_state=13)
+
+OLS_report_and_ANOVA(summer_data, 'pustynia', 'P/PET')
+
+perform_Kruskal_Wallis(summer_data, 'pustynia', 'P/PET', 13)
+
+winter_data = December_2021_DF.drop(columns=['Month', 'Year'])
+
+test_for_residuals_normality(winter_data, 'Rainf')
+
+plot_density(winter_data, 'pustynia', 'Rainf', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'Rainf')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'Rainf', 13)
+
+test_for_residuals_normality(winter_data, 'Evap')
+
+plot_density(winter_data, 'pustynia', 'Evap', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia','Evap')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia','Evap', 13)
+
+test_for_residuals_normality(winter_data, 'AvgSurfT')
+
+plot_density(winter_data, 'pustynia', 'AvgSurfT', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'AvgSurfT')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'AvgSurfT', 13)
+
+test_for_residuals_normality(winter_data, 'Albedo')
+
+plot_density(winter_data, 'pustynia', 'Albedo')
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'Albedo')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'Albedo', 13)
+
+test_for_residuals_normality(winter_data, 'SoilT_40_100cm')
+
+plot_density(winter_data, 'pustynia', 'SoilT_40_100cm', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'SoilT_40_100cm')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'SoilT_40_100cm', 13)
+
+test_for_residuals_normality(winter_data, 'GVEG')
+
+plot_density(winter_data, 'pustynia', 'GVEG', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'GVEG')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'GVEG', 13)
+
+test_for_residuals_normality(winter_data, 'PotEvap')
+
+plot_density(winter_data, 'pustynia', 'PotEvap', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'PotEvap')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'PotEvap', 13)
+
+test_for_residuals_normality(winter_data, 'RootMoist')
+
+plot_density(winter_data, 'pustynia', 'RootMoist', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'RootMoist')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'RootMoist', 13)
+
+test_for_residuals_normality(winter_data, 'SoilM_100_200cm')
+
+plot_density(winter_data, 'pustynia', 'SoilM_100_200cm', random_state=13)
+
+OLS_report_and_ANOVA(winter_data, 'pustynia', 'SoilM_100_200cm')
+
+perform_Kruskal_Wallis(winter_data, 'pustynia', 'SoilM_100_200cm', 13)
+
+full_summer_data = spark.sql('''SELECT * from nasa_full WHERE Month = 7 AND (Year = 2021 OR Year = 2019)''').toPandas()
+full_summer_data = full_summer_data.drop(columns=['Month', 'Year', 'lon', 'lat'])
```

Comment:
Po co robić `SELECT *` by zaraz zrobić drop. No i ten toPandas() po środku. To już sensowniej jest w tej kolejności
```suggestion
full_summer_data = spark.sql('''SELECT * from nasa_full WHERE Month = 7 AND (Year = 2021 OR Year = 2019)''').drop('Month', 'Year', 'lon', 'lat').toPandas()
```

Reply:
Ok, zmieniłam.